src/app/server/Server.cpp

```diff
@@ -197,6 +197,8 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     err = chip::app::InteractionModelEngine::GetInstance()->Init(&mExchangeMgr);
     SuccessOrExit(err);
 
+    chip::Dnssd::Resolver::Instance().Init(DeviceLayer::UDPEndPointManager());
```

Comment:
This is already done in OTA Requestor... should it be removed there? https://github.com/project-chip/connectedhomeip/blob/master/examples/ota-requestor-app/linux/main.cpp#L184

Reply:
Yes, I'd think `Server.cpp` is the right home for this bit of logic.

Reply:
Maybe as a followup? I believe I had added that bit to make CI pass, however it seems obvious now that this needs to be more generic (i.e. all servers have it) instead of just for the requestor app.